keymaps/pdf-view.json

```diff
@@ -6,6 +6,7 @@
   },
 
   ".platform-darwin .pdf-view": {
+    "alt-cmd-=": "pdf-view:zoom-fit",
```

Comment:
There's already something bound to `alt-cmd-=` on macos -- the `pane:increase-size` command. Just wondering if you intentionally picked this and if not -- is there some other keybinding we can pick instead? 💭

Reply:
No, just a random choice; I change it to cmd/ctrl-atl-9.